src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java

```diff
@@ -564,8 +569,15 @@ public void testGetCoursesWithQuizExercises() throws Exception {
     // Test
     public void testGetCourseForDashboard() throws Exception {
         List<Course> courses = database.createCoursesWithExercisesAndLecturesAndLectureUnits(true, false);
+
+        hibernateQueryInterceptor.startQueryCount();
+
         Course receivedCourse = request.get("/api/courses/" + courses.get(0).getId() + "/for-dashboard", HttpStatus.OK, Course.class);
 
+        // Checks the amount of DB queries done for the rest call. Needs to be done right after the rest call to avoid counting queries during the setup
+        // Ensures that the amount of queries done doesn't increase
```

Comment:
This is now explained in the documentation, so I would leave out the comment here